libraries/core-react/CHANGELOG.md

```diff
@@ -5,6 +5,35 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.5] - 2021-02-22
+
+### Added ✨
+
+- Progress indicators ([#1090](https://github.com/equinor/design-system/issues/1090), [#1098](https://github.com/equinor/design-system/issues/1098))
+  - `Circular`, `Star` and `Dots` now have a `size` property for adjusting size.
+  - `Circular` and `Dots` now have a `color` property following the same color scheme as our other components.
+  - Added compound component support. You can now find the progress indicators as `Progress.Dots`, `Progress.Star` etc..
```

Comment:
```suggestion
  - Refactored as compound components. You can now find the progress indicators as `Progress.Dots`, `Progress.Star` etc..
```

Reply:
Since we’re not actually _supporting_ compound components, but offering them…

Reply:
@vnys We still have the individual sub-components being exported as `LinearProgress`, `CircularProgress` etc, so maybe also we should mention that they will be removed in the future

Reply:
Yes, I agree

Reply:
Updated: 

>     - The individual `LinearProgress`, `CircularProgress`, `StarProgress` & `DotProgress` will be removed at a later time